packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -62,6 +62,11 @@ function plainFunction(path: NodePath, callId: Object) {
 
   node.async = false;
   node.generator = true;
+  // Either the wrapped generator is invoked with `.apply(this, arguments)` or it has no params,
+  // so it should capture `arguments`
+  if (node.shadow) {
+    node.shadow = { arguments: false };
```

Comment:
Is there any danger that this was already an object instead of `true` and this is blowing away shadowing flags for `this`? My background on shadowing is pretty rusty.